### ISSUE TITLE
fix: 🐛 syn-combobox: Keyboard navigation not working after text was entered (only in wrappers)

### ISSUE THREAD

**Issue by schilchSICKAG** [synergy-design-system/synergy-design-system#632]
# Checklist (for the issue creator)

- [x] I have verified the bug exists in the latest version of `@synergy-design-system/components` (e.g. via `pnpm install @synergy-design-system/components`).
- [x] I can provide a simple, reproducible example of the issue (e.g. via code-sandbox or sample code, ...).
- [x] I would be willing to provide a PR for this issue.

# Version

The bug occurred with the following version of the component library: 2.8.0 (only in framework wrappers, not in the native web-component)

# Component

The issue arises when using the following component(s): syn-combobox

# Environment

The issue occurred with the following operating system(s): All
The issue occurred in the following browser(s): All

# Description

When pressing key down in a `<syn-combobox>`, it will automatically open the associated dropdown and let the user select one of the entries via keyboard. However, this does not work anymore when the user entered at least one letter into the input field. There is no keyboard navigation happening anymore in this case.

# Acceptance criteria

- [ ] All DoD items successfully completed
- [ ] Keyboard navigation should work regardless if the user has entered something or not.
